app/assets/javascripts/Patient.js

```diff
@@ -0,0 +1,162 @@
+var Schema = mongoose.Schema;
+var Code = Schema.Types.Code;
+var Interval = Schema.Types.Interval;
+var Quantity = Schema.Types.Quantity;
+
+var PatientSchema = new Schema({
```

Comment:
checking hds vs this.

title
religious affiliation
effective time
languages
test id
bundle id
marital status/record number/record assigner

does it make sense to include any of these? I lean to no, other than maybe bundle id. Does cypress have thoughts on bundle id?